processing/src/main/java/io/druid/query/extraction/JavascriptExtractionFn.java

```diff
@@ -115,6 +115,12 @@ public boolean preservesOrdering()
   }
 
   @Override
+  public ExtractionType getExtractionType()
+  {
+    return ExtractionType.MANY_TO_ONE;
```

Comment:
There are some cases where `ONE_TO_ONE` would make sense, but I think that could be separate from this PR. Having this as the default continues current behavior.